http_server/simple_led_http.py

```diff
@@ -0,0 +1,54 @@
+from amaranth import Module
+from amaranth.lib.wiring import In, Out, Component
+from amaranth.lib import stream
+
+
+class SimpleLedHttp(Component):
+    """
+    SimpleLedHttp accepts an HTTP/1.0 request to change LED colors.
+
+    Returns an HTTP OK status if accpeted.
+
+    Expects a POST to the /led path with a body containing 8 hex
+    characters corresponding to red, green, and blue LED values.
+
+    Attributes
+    ----------
+    request: Signal(1), in

```

Comment:
Do some of these inputs naturally map to a `Signature`? e.g. is `request,input,output,complete` something we're going to want to use for multiple modules?

Example: [here](https://github.com/cceckman/http-accel/blob/1c3b4da650f6c3312d938e27337c77042e144d71/almost_tcp/message_hdl.py#L76) is a signature I'm trying to use for transmitting a packet-at-a-time around.